codalab/apps/newsletter/models.py

```diff
@@ -0,0 +1,33 @@
+import hashlib
+import json
+
+import requests
+from django.db import models
+from django.db.models.signals import pre_delete
+from django.dispatch import receiver
+
+from codalab import settings
+
+
+class NewsletterUser(models.Model):
```

Comment:
Maybe a better name would be `NewsletterSubscription` since this isn't really a user object?

Reply:
Updated in latest push